### ISSUE TITLE
todoist module panics when trying to clear or delete task in empty project

### ISSUE THREAD

**Issue by kognate** [wtfutil/wtf#578]
## What's broken?

When using the todoist module, if there is an empty project loaded and you type `c` or `d` (clear and delete respectively)  you get a crash:

```panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc0005b1000)
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190721135419-23dc8a0944e4/application.go:149 +0x82
panic(0x191db00, 0x2f8fbb0)
	/usr/local/Cellar/go/1.12.7/libexec/src/runtime/panic.go:522 +0x1b5
github.com/wtfutil/wtf/modules/todoist.(*Project).currentTask(...)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/todoist/project.go:63
github.com/wtfutil/wtf/modules/todoist.(*Project).deleteSelectedTask(0xc00086ac80)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/todoist/project.go:79 +0x10a
github.com/wtfutil/wtf/modules/todoist.(*Widget).Delete(0xc0003fe2c0)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/todoist/widget.go:118 +0x4d
github.com/wtfutil/wtf/view.(*KeyboardWidget).InputCapture(0xc0003fe2c0, 0xc0006c3040, 0xc0007bdd90)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/view/keyboard_widget.go:85 +0xf1
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0006c3040, 0xc0005222f0)
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190721135419-23dc8a0944e4/box.go:157 +0x75
github.com/rivo/tview.(*Application).Run(0xc0005b1000, 0x0, 0x0)
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190721135419-23dc8a0944e4/application.go:234 +0x396
main.main()
	/Users/ccummer/go/src/github.com/wtfutil/wtf/main.go:74 +0x288
```

**Comment by kognate** [comment]
See also #588

**Comment by senorprogrammer** [comment]
Fixed by #588
